.eslintrc

```diff
@@ -70,14 +70,22 @@
         "requireReturn": false
       }
     ],
-    "require-jsdoc": [
+    "require-jsdoc-except/require-jsdoc": [
       "error",
       {
         "require": {
           "MethodDefinition": true,
           "ClassDeclaration": true,
           "FunctionDeclaration": true
-        }
+        },
+        "ignore": [
+          "constructor",
```

Comment:
Do we have another place where we can document the params that you pass into a constructor?

Reply:
The description for the constructor can just be "Constructor" but I think the params is the important part. Especially if it's overloaded.

Reply:
The interesting idea, but now the params is props, without any specification. There is a plan to use [airbnb eslint|https://www.npmjs.com/package/eslint-config-airbnb] in future.

Reply:
Sounds good.  We have the eslint settings that Yaroslav started. But we didn't implement that for all the repos (I think), but that's a separate discussion.